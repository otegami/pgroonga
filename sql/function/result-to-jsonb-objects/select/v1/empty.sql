SELECT pgroonga_result_to_jsonb_objects(
  '[
     [0, 0.0, 0.0],
     [
       [
         [0],
         [["_id", "UInt32"]]
       ]
     ]
   ]'::jsonb);
