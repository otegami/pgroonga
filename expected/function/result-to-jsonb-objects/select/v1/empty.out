SELECT pgroonga_result_to_jsonb_objects(
  '[
     [0, 0.0, 0.0],
     [
       [
         [0],
         [["_id", "UInt32"]]
       ]
     ]
   ]'::jsonb);
 pgroonga_result_to_jsonb_objects 
----------------------------------
 []
(1 row)

